front/style/pages/_profile.scss

```diff
@@ -134,3 +134,9 @@
         }
     }
 }
+
+#adherent_email_subscription_emails_subscriptions {
```

Comment:
We don't generally use IDs in CSS. Need to use a class. Also if it's just a margin, you can use one of our reusable classes.